tools/clang/test/CodeGenSPIRV/intrinsics.dot4add.hlsl

```diff
@@ -0,0 +1,14 @@
+// Run: %dxc -E frag -T ps_6_4 -fspv-target-env=vulkan1.1
+
+uint frag(float4 vertex
+          : SV_POSITION) : SV_Target {
+  uint acc = 0;
+
+  // CHECK: 8:14: error: dot4add_u8packed intrinsic function unimplemented
+  acc += 1 + dot4add_u8packed(vertex.x, vertex.y, uint(vertex.y));
```

Comment:
Could you please add
```
acc += dot4add_u8packed(vertex.x, vertex.y, uint(vertex.y)) + 1;
acc = 1 + dot4add_u8packed(vertex.x, vertex.y, uint(vertex.y));
acc = dot4add_u8packed(vertex.x, vertex.y, uint(vertex.y)) + 1;
```
?

Since the main issue is the `lhs` and `rhs` and the type of binary operator, I want to know what happens when `lhs` has the issue and it is not compound assignment op.

Reply:
Done